models/modelling/olids/observations/int_eosinophil_percentage.sql

```
@@ -79,10 +69,10 @@ SELECT
     is_negative,
     is_extreme_outlier,
     CASE
-        WHEN inferred_value IS NULL OR confidence = 'NONE' THEN 'Abnormal'
+        WHEN inferred_value IS NULL THEN 'Abnormal'
```

SQLFluff (4.2.2)

[error] Boolean/null literals must be lower case.

(CP04)